environment.yml

```
@@ -59,7 +58,7 @@ dependencies:
   - importlib_resources
   # docs
   - pygments >=1.5
-  - sphinx
+  - sphinx >=4.1.0
```

My mistake, sphinx actually dropped support in `2.0.0`, there is no need to pin anything as docutils hasn't been built for Python 2 for a long time:
```suggestion
  - sphinx
```